packages/components/src/components/primitives/Textarea/Textarea.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { screen } from '@testing-library/react';
+import { Textarea } from './Textarea';
+
+const testId = 'cmpsr.textarea';
+
+describe('Textarea', () => {
+  test('should render default textarea', () => {
+    renderWithProviders(<Textarea />);
+    screen.getByTestId(testId);
+  });
+
+  test('should render with value', () => {
+    const value = 'Filled text input area...';
+    renderWithProviders(<Textarea defaultValue={value} />);
+    const textarea = screen.getByTestId(testId);
+    expect(textarea).toHaveValue(value);
+  });
+
+  test('should render an invalid textarea', () => {
+    renderWithProviders(<Textarea isInvalid />);
+    const textarea = screen.getByTestId(testId);
+    expect(textarea).toHaveStyle({
+      borderColor: `var(--chakra-colors-alert-error-default)`,
+    });
+  });
```

Comment:
no need to unit test styles, we have chromatic for this

Reply:
I think we should be good to remove `import '@testing-library/jest-dom';` then

Reply:
`@testing-library/jest-dom` can make it easier for us how we validate what is rendered, https://github.com/testing-library/jest-dom#the-problem, but obviously it is one more option